io.github.jkozera.ZevDocs.json

```diff
@@ -0,0 +1,74 @@
+{
+    "app-id" : "io.github.jkozera.ZevDocs",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.golang"
+    ],
+    "command" : "zevdocs",
+    "rename-icon": "zevdocs",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        "--filesystem=host:ro",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O2",
+        "cxxflags" : "-O2",
+        "env" : {
+            "V" : "1"
+        },
+        "build-args": ["--share=network"]
```

Comment:
This won't pass. Build needs to work offline for predictable rebuilds. As you're probably using it for Go part, it would be better to just vendor all dependencies in the repository.

Reply:
Thanks for the quick feedback! I'm not sure if I like vendoring, IMHO it's worth keeping the repo clean from dependencies, and instead use tools like `dep` with `Gopkg.lock` file which achieve similar reproducability.

How about we put all the dependencies from the `Gopkg.lock` into the json, as per the new commit?

I'll update appdata shortly.

Reply:
There are already some helpers available in [flatpak-builder-tools](https://github.com/flatpak/flatpak-builder-tools) but not for Go unfortunately.